### PR TITLE
ELPP-3458: Preloadable assets

### PR DESCRIPTION
**Pull request by davidcmoulton**
Adds a `preload.json` file to define assets for preloading.

Also moves `@font-face` declarations into scss.

### CODE REVIEW

File: assets/preload.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "resources": [
+    {
+      "type": "font/woff2",
```
Comment (thewilkybarkid) [inline assets/preload.json]: Include `as` too.

File: assets/preload.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (thewilkybarkid) [inline assets/preload.json]: Can just be a list of objects?

File: assets/preload.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "resources": [
+    {
+      "type": "font/woff2",
+      "paths": "fonts/NotoSans-Regular-webfont-custom-2-subsetting.woff2"
```
Comment (thewilkybarkid) [inline assets/preload.json]: `path`?

File: assets/preload.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+      "type": "font/woff2",
+      "paths": "fonts/NotoSans-SemiBold-webfont-custom-2-subsetting.woff2"
+    },
+    {
+      "type": "font/woff2",
+      "paths": "fonts/NotoSerif-Regular-webfont-custom-2-subsetting.woff2"
+    },
+    {
+      "type": "font/woff2",
+      "paths": "fonts/NotoSerif-Bold-webfont-basic-latin-subsetting.woff2"
```
Comment (thewilkybarkid) [inline assets/preload.json]: #862 wasn't going to preload this.

File: assets/preload.json (hunk 1 above)
Comment (davidcmoulton) [reply assets/preload.json]: Can the `as` value be reliably derived from the `type`?

File: assets/preload.json (hunk 4 above)
Comment (davidcmoulton) [reply assets/preload.json]: Thanks, rushing.

File: assets/preload.json (hunk 2 above)
Comment (davidcmoulton) [reply assets/preload.json]: Yup. Old habits worrying about JSON hijacking, will update.

File: assets/preload.json (hunk 1 above)
Comment (thewilkybarkid) [reply assets/preload.json]: In the event we start including other things might become tricky. Not sure we would, but isn't onerous to just set it here.

File: assets/preload.json (hunk 3 above)
Comment (thewilkybarkid) [reply assets/preload.json]: Even `uri`, for foreign resources?

Comment (thewilkybarkid) [reply assets/preload.json]: Maybe ignore for now.

**Comment by thewilkybarkid** [comment]
Can remove `WIP`?

**Pull request merged**